src/main/java/at/hannibal2/skyhanni/features/rift/area/colosseum/BactePhase.kt

```diff
@@ -0,0 +1,104 @@
+package at.hannibal2.skyhanni.features.rift.area.colosseum
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.mob.Mob
+import at.hannibal2.skyhanni.events.LorenzChatEvent
+import at.hannibal2.skyhanni.events.LorenzRenderWorldEvent
+import at.hannibal2.skyhanni.events.MobEvent
+import at.hannibal2.skyhanni.events.SecondPassedEvent
+import at.hannibal2.skyhanni.features.rift.RiftAPI
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.RegexUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.RenderUtils.drawDynamicText
+import at.hannibal2.skyhanni.utils.getLorenzVec
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+@SkyHanniModule
+object BactePhase {
+
+    private val group = RepoPattern.group("rift.colosseum.bacte")
+    /**
+     * REGEX-TEST: §2﴾ §8[§7Lv10§8] §l§aBa§r §a800§f/§a1,000§c❤ §2﴿
+     */
+    private val namePattern by group.pattern(
+        "name",
+        "§2﴾ §8\\[§7Lv\\d+§8\\] §l§a(?<name>.*)§r §.[\\d.,]+§f\\/§a[\\d.,]+§c❤ §2﴿",
```

Comment:
Redundant character escape '\\]' in RegExp 
Redundant character escape '\\/' in RegExp

Reply:
<img width="762" alt="image" src="https://github.com/user-attachments/assets/b91dd9a6-dab5-4267-9fbb-3b191b8c47d7">
we have so many of these in the code, I don't think it really matters

Reply:
i guess